Changes.md

```diff
@@ -1,3 +1,11 @@
+0.58.x.x (relative to 0.58.3.2)
+========
+￼
+Fixes
+-----
+￼
+- ImageReader : Fix fileName being evaluated in multiple contexts
```

Comment:
- ImageReader : Fixed bug which caused `fileName` to be evaluated in multiple contexts.